src/hf_hydrodata/data_catalog.py

```diff
@@ -656,6 +662,75 @@ def _is_row_match_options(row: ModelTableRow, options: dict) -> bool:
     return result
 
 
+def _get_point_citations(dataset):
+    """
+    Return a dictionary with relevant citation information.
+
+    Parameters
+    ----------
+    dataset : str
+        Source from which requested data originated. Currently supported: 'usgs_nwis', 'snotel',
+        'scan', 'ameriflux', 'jasechko_2024', 'fan_2013'.
+
+    Returns
+    -------
+    str
+        String containing overall attribution instructions for the provided dataset.
+    """
+    try:
+        assert dataset in [
+            "usgs_nwis",
+            "snotel",
+            "scan",
+            "ameriflux",
+            "jasechko_2024",
+            "fan_2013",
+        ]
+    except:
+        raise ValueError(
+            f"Unexpected value of dataset, {dataset}. Supported values include 'usgs_nwis', 'snotel', 'scan', 'ameriflux', 'jasechko_2024', and 'fan_2013'"
+        )
+
+    if dataset == "usgs_nwis":
```

Comment:
This is a good point - I agree that this could be incorporated into the data catalog, especially now that each dataset just has associated text (we used to do some fancier things looking up site IDs, etc.). Let me add this to backlog!

Reply:
Cool, thanks! And thanks for the past context